protobuf/model_config.proto

```diff
@@ -1869,6 +1869,77 @@ message ModelResponseCache
   bool enable = 1;
 }
 
+//@@
+//@@  .. cpp:var:: message ModelMetrics
+//@@
+//@@     The metrics setting of this model.
+//@@
+message ModelMetrics
+{
+  //@@
+  //@@  .. cpp:var:: message MetricControl
+  //@@
+  //@@     Override metrics settings of this model.
+  //@@
+  message MetricControl
+  {
+    //@@
+    //@@  .. cpp:var:: message MetricIdentifier
+    //@@
+    //@@     Specify metrics to be overridden with metric_option.
+    //@@
+    message MetricIdentifier
+    {
+      //@@  .. cpp:var:: string family
+      //@@
+      //@@     The name of the metric family to override
+      //@@     with the custom value.
+      //@@
+      string family = 1;
+    }
+
+    //@@  .. cpp:var:: message HistogramOptions
+    //@@
+    //@@     Histogram metrics options.
+    //@@
+    message HistogramOptions
+    {
+      //@@  .. cpp:var:: double buckets (repeated)
+      //@@
+      //@@     Repeated double type for histogram bucket boundaries.
+      //@@
+      repeated double buckets = 1;
```

Comment:
Does user need to specify 0.0?

Reply:
Short answer no. "0" can be parsed as double. I tested [ -1, 0.0, 1, 2.5 ] buckets but somehow did not check it in. Odd.

Reply:
Updated test